presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba4.result

```diff
@@ -1,5 +1,5 @@
 -- delimiter: |; ignoreOrder: false; ignoreExcessRows: false;
 hive|default|nation|n_nationkey|-5|bigint|19|8|0|10|1|null|null|null|null|null|1|YES|null|null|null|null|null|
-hive|default|nation|n_name|12|varchar|2048|2048|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|
+hive|default|nation|n_name|12|varchar(25)|25|25|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|
```

Comment:
That is coming from Presto.  We see the same value with the open source JDBC driver.  I'm not sure why it was changed from `varchar`, but my reading of the docs is that either value is valid:

[JDBC spec](https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getColumns(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String)) says:
> TYPE_NAME String => Data source dependent type name, for a UDT the type name is fully qualified
> COLUMN_SIZE int => column size.